app/screens/home/home_screen.tsx

```diff
@@ -8,17 +8,47 @@ import { BottomBar } from "@/components/navigation/BottomBar";
 import { DBChallenge } from "@/firebase/FirestoreCtrl";
 import { getAuth } from "firebase/auth";
 import { ThemedText } from "@/components/theme/ThemedText";
+import { ChallengeDescription } from "@/components/home/Challenge_Description";
+import { DBChallengeDescription } from "@/firebase/FirestoreCtrl";
 
 // Get the screen dimensions
 const { width, height } = Dimensions.get("window");
 
 export default function HomeScreen({ user, navigation, firestoreCtrl }: any) {
   const [challenges, setChallenges] = useState<DBChallenge[]>([]);
+  const [currentChallenge, setCurrentChallenge] =
+    useState<DBChallengeDescription>({
+      title: "Challenge Title",
+      description: "Challenge Description",
+      endDate: new Date(2024, 1, 1, 0, 0, 0, 0),
+    });
```

Comment:
There is confusion between the const challenges and this new const currentChallenge, maybe you should rename one of them.